Dockerfile.aarch64

```diff
@@ -39,6 +39,7 @@ RUN apt-get update && apt-get install -y \
 	libcap-dev \
 	libltdl-dev \
 	libsystemd-dev \
+	libyaml-dev \
```

Comment:
Good question. I am not a Python expert, but when I added yamllint to the Dockerfile, there were some complaints in the output that libyaml could not be found during compilation. However, I did not check with the other Dockerfiles to see if it's used there.